### PR TITLE
Blocks for HTML mode

### PR DESCRIPTION
**Pull request by sakagg**
Adds support for block editing in HTML Language in PencilCode
To test it simply navigate to `user.pencilcode.net.dev/edit/foo.html`

### CODE REVIEW

File: content/src/view.js [hunk 1]
```diff
@@ -1600,6 +1604,19 @@ function updatePaneTitle(pane) {
       label = 'data';
     } else if (/^text\/html/.test(paneState.mimeType)) {
```
Comment (davidbau) [inline content/src/view.js]: Let's move this HTML case inside the "else" below - it looks like the only difference is `label = 'html'` vs `'code'`, so let's just put the html `if` around that one small difference.

Comment (davidbau) [inline content/src/view.js]: That will also fix the bug where `textonly = false` is getting missed - which causes you to currently lose the yellow tab block toggle button.

**Comment by davidbau** [comment]
right now you haven't added droplet support to `setupSubEditor`, which is what would need to change if you want to show block mode in the split-pane JS/HTML/CSS view.  That's OK for this first PR, I think - but it's something you should consider adding for the future.

File: content/src/view.js (hunk 1 above)
Comment (sakagg) [inline content/src/view.js]: @davidbau Fixed it.

**Comment by sakagg** [comment]
I was thinking that in the case of split-pane, the work area for HTML or CSS is not too much and blocks can look cramped up. Although, this is how most people will use the HTML mode, so we have to provide block support here as well. Maybe we can make the palette smaller, or something like that.

**Comment by davidbau** [comment]
OK.  One final thing: can you find a way to add a simple test for block mode in HTML?  It would be enough to verify that both block mode and text mode can show for HTML.

**Comment by sakagg** [comment]
I've added the tests. Anything else?

**Comment by davidbau** [comment]
LGTM!

**Pull request merged**